.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: CI/CD Workflow
```

Comment:
build-publish-next or something describing what is done in the workflow

Reply:
renamed to "Build & Publish `next`"